src/main/webapp/mentors.js

```diff
@@ -1,21 +1,26 @@
 import standardizeFetchErrors from '/js/fetch_handler.js';
-getMentors();
-let form = document.getElementById('mentor-form');
-form.addEventListener('submit', submitForm);
+
+if (window.location.href.includes('mentors.html')) {
+  getMentors();
+}
+if (window.location.href.includes('mentor_form.html')) {
+  console.log('mentor form page!');
+  document.getElementById('mentor-form').addEventListener('submit', submitForm);
+}
```

Comment:
This is a really interesting piece of code! I'm going to add some functionality to the dashboard with this!

Reply:
Yeah, it's interesting. I wonder if it makes sense to just call both functions, and have the functions determine whether they should even do anything. It seems possible since they both rely on the presence of an element with a certain ID, right? 

So in the beginning of both functions, we could just check if that element exists, and return if not.

Reply:
Would slightly prefer that since otherwise this is a little brittle; it'll break if we move the pages to different files.

Reply:
Yeah I agree with Bill here. Or we could move general mentor JS to a common file like `mentors.js`, and then move page-specific stuff to individual files that are only loaded on their respective pages. That way we don't have to worry about differentiating between pages but still can reuse the mentor code.

Reply:
Sounds good! I will check for existence instead of checking which page I am on.